src/index.js

```diff
@@ -0,0 +1,57 @@
+require('dotenv').config();
+
+const debug = require('debug')('supersonic-lobster:index');
+const merry = require('merry');
+const fs = require('fs');
+const path = require('path');
+const index = require('./routes/index');
+const ocSsr = require('./middlewares/oc-ssr');
+
+const app = merry();
+
+app.route('GET', '/', (req, res, ctx) => {
+  ocSsr({
+    ocRegistryBaseUrl: process.env.OC_REGISTRY_BASE_URL,
+    apiKey: process.env.NASA_API_KEY,
+    trackingId: process.env.TRACKING_ID
+  })(req, res, () => {
+    index({ ocRegistryBaseUrl: process.env.OC_REGISTRY_BASE_URL })(
+      req,
+      res,
+      ctx
+    );
+  });
+});
+
+app.route('GET', '/favicon.ico', (req, res) => {
+  const headers = { 'Content-Type': 'image/x-icon' };
+  res.writeHead(301, headers);
+  res.end();
+});
+
+app.route('GET', '/images/FFFFFF-0.png', (req, res) => {
+  const contents = fs.readFileSync(
+    path.join(__dirname, 'public/images/FFFFFF-0.png')
+  );
+  const headers = {
+    'Content-Type': 'image/png',
+    'Content-Length': contents.length
+  };
+  res.writeHead(200, headers);
+  res.end(contents, 'binary');
+});
+
+app.route('GET', '/style.css', (req, res, ctx) => {
+  const contents = fs.readFileSync(path.join(__dirname, 'public/style.css'));
+  const headers = { 'Content-Type': 'text/css' };
+  ctx.send(200, contents.toString(), headers);
```

Comment:
Hey Mattia, I am sorry I've been really quite busy with work this week; leaves of less time for extracurriculars.

I think we've built merry to be flexible enough to do whatever it is you need to do server side. API is an API, it doesn't really matter whether it serves a website or just JSON. Most of the server-side stuff I built for any purpose serves JSON anyways; so it's one and the same for me.

The general example we have on top of the readme could be used in both cases, imo. If you need further examples, Yosh's built out an [analytics-service](https://github.com/shipharbor/analytics-service/blob/master/index.js). If you run into bugs, probably best to use the issue tracker ✌🏽

Thanks!